master/usage/migrating-pools.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Changing IP pools
+---
+
+## About changing IP pools
+
+When using {{ site.prodname }} IPAM, each workload is assigned an address from the selection of configured IP pools.
+There are a few reasons you may want to change the IP pool being used on a running cluster, including moving to a
+larger CIDR which can accomodate more workloads, or moving off of a CIDR that was used accidentally.
+
+### Purpose of this page
+
+Provide guidance on how to change from one IP pool to another on a running cluster.
```

Comment:
I think we could profitably lose lines 11-13, but if you feel attached to them, please feel free to retain them.

Reply:
Hmm. I was trying to follow the format of this page: https://docs.projectcalico.org/master/usage/decommissioning-a-node

I can see how it bloats up the page but I also think there needs to be a transition from "background" to the "procedure"...